guacamole/src/main/frontend/src/app/clipboard/directives/guacClipboard.js

```diff
@@ -92,17 +141,23 @@ angular.module('clipboard').directive('guacClipboard', ['$injector',
         // that indicate the clipboard field may have been changed
         element.addEventListener('input', updateClipboardData);
         element.addEventListener('change', updateClipboardData);
+        element.addEventListener('focus', displayAndUpdateClipboardEditor);
+        element.addEventListener('blur', hideAndUpdateClipboardEditor);
```

Comment:
Do we really want to hide the contents of the clipboard every time it's blurred? That would get pretty annoying. I'd think that just initially hiding the clipboard every time the sidebar is opened and re-hiding when the sidebar is closed should be good enough. 

I'd welcome other opinions on this, however.

Reply:
This is how I originally had planned to do it, but I felt like it's more secure to just hide whenever it isn't focused in case of password copying

Reply:
I don't think there's any one single correct answer for this one - I could see some users (like me) getting frustrated when the clipboard contents keep getting hidden while working in a connection, making it more difficult to keep track of the clipboard content.

On the other hand, I could see why some users might want to automatically rehide every time, in case they copy a password locally and have it get populated into their clipboard in the background without realizing it.

I almost wonder if this should be a configuration setting...

@mike-jumper thoughts?